### ISSUE TITLE
Scoot doesn't find elements in certain apps (e.g. Chrome, Firefox, anything built with Electron) unless VoiceOver is enabled

### ISSUE THREAD

**Issue by mjrusso** [mjrusso/scoot#11]
Ideally, Scoot should be able to find all elements on the screen, even if VoiceOver is off.

Relevant resources:

- https://www.electronjs.org/docs/latest/tutorial/accessibility
- https://github.com/electron/electron/issues/7206
- https://github.com/electron/electron/pull/10305
- https://www.chromium.org/developers/design-documents/accessibility#TOC-How-Chrome-detects-the-presence-of-Assistive-Technology
- https://bugzilla.mozilla.org/show_bug.cgi?id=1664992

This code sample also looks handy: https://github.com/rxhanson/Rectangle/blob/master/Rectangle/AccessibilityElement.swift

**Comment by mjrusso** [comment]
Something to try: before bringing Scoot to the foreground, set the `AXManualAccessibility` and/or `AXEnhancedUserInterface` attribute(s) on the focused window of the frontmost app. (Also try removing the attribute(s) when bringing Scoot to the background.)

**Comment by mjrusso** [comment]
Implemented in 46f764f. Note the FIXMEs I documented in the code:

>  FIXME: an app may take some time to enable accessibility APIs; i.e., we may query the element tree too quickly after setting the `AXEnhancedUserInterface` attribute. (This seems to be especially prevalent with Electron-based apps.) For the time being, as a workaround, users can simply re-invoke Scoot to force the element tree to be traversed again.

And: 

>  FIXME: remove the `AXEnhancedUserInterface` attribute, for example when Scoot is moved to the background. (But only if Scoot added it in the first place.)

**Comment by mjrusso** [comment]
I've changed this behaviour in 2e6bd2f4b6164052c4366da0889cea943f44b5ff, because the race condition with the previous implementation (resulting in Scoot having an incomplete picture of the accessibility tree) was annoying. The new approach is straightforward, simply turning on `AXEnhancedUserInterface` for all apps (those that are running when Scoot is launched, and also for any apps that are launched after Scoot).
